components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/services/UserInformationRecoveryService.java

```diff
@@ -70,6 +70,7 @@
  */
 public class UserInformationRecoveryService {
 
+    private static final String USER_ALREADY_EXISTING = "UserAlreadyExisting";
```

Comment:
It will be more meaningful, if the constant is declared in user.mgt package in common constant declaration and re-used here. This is becuase if anyone later change this value in user.mgt package, he/she should remember to come and modify it here too.